client/posts/render/body.ts

```diff
@@ -481,6 +481,16 @@ function parseCommand(bit: string, { commands, state }: PostData): string {
             inner = commands[state.iDice++].val ? "flap" : "flop"
             break
         case "8ball":
+        case "pyu":
+            if (boardConfig.pyu) {
+                inner = commands[state.iDice++].val.toString()
+                break
```

Comment:
Consider the following: What happens, if pyu is matched, but configs are disabled.

Reply:
It should do nothing and fallthrough to the next case just like it has before this commit, which is what that should be doing already.

Reply:
It will format it as a valid command. Check the last return.

Reply:
Ok, I keep trying to find what you mean but I just can't see it.
Could you give me some more information?
Do I need to break after checking for the pyu in boardconfig or do I need to do something with the "rcount" case?
is it somehow formatting the if statement?
Do I need to create a different switch statement for bit and move the commands over there?